docs/SMCWG-charter.md

```diff
@@ -77,21 +77,27 @@ capitalized terms in this Charter.
 The authorized scope of the SMCWG SHALL be to discuss, adopt, and maintain
 policies, frameworks, and sets of standards related to the issuance and
 management of S/MIME certificates by third-party CAs under a publicly trusted
-root, limited as follows:
+root.
 
-- Baseline verification of control over email addresses, including those used
-  by a natural person or a legal entity, or used by automated systems such as
-  for mailing lists
-- Identity validation for natural persons and legal entities
+The primary deliverable shall be scoped as follows:
```

Comment:
I appreciate this attempt, but I don't think it's helpful to adopt language about a "primary deliverable", which has no definition or precedent.  If you want to go down this road, additional language would be necessary to make it clear that topics outside of the primary deliverable aren't out of scope.  I realize that deleting "limited as follows" attempts to do that, but I don't think it's sufficient given the history of this topic.

Reply:
Precedent exists, within the Network Security WG (later, rechartered to Subcommittee), which was directed to a specific task for a particular time.

The ambiguity is resolved by Section 7; they go hand in hand.